ArchiSteamFarm/Core/Statistics.cs

```diff
@@ -407,7 +374,7 @@ internal sealed class Statistics : IAsyncDisposable {
 			return (false, false);
 		}
 
-		ImmutableHashSet<ListedUser>? listedUsers = await GetListedUsers().ConfigureAwait(false);
+		ImmutableHashSet<ArchiNet.ListedUser>? listedUsers = await ArchiNet.GetListedUsers(Bot.ArchiWebHandler.WebBrowser).ConfigureAwait(false);
```

Comment:
The other calls just pass `Bot` instead of `Bot.ArchiWebHandler.WebBrowser`. `ArchiNet`s concern is how to get the listed users. `Statistics`' concern is what do do with them. Therefore the choice which `WebBrowser`-instance to use should be the one of `ArchiNet`.

Reply:
You miss a point, this call is done on per-bot basis, and the web browser passing here is in order to glue the request to given bot, for example for logging purposes in case of an error.

It's not for passing a reference to web browser, bot-less contexts (like fetching checksum) do not ask for a web browser.

Reply:
I didn't want you to remove the parameter. I understand why it is there. I wanted you to change it to the `Bot`-instance itself for `ArchiNet` to do the `.ArchiWebHandler.WebBrowser` part.

Reply:
OK, that can be done indeed, even though it's a bit of a burden as we do not require whole instance.